### ISSUE TITLE
SmolVLA Gazebo Simulation Issue

### ISSUE THREAD

**Issue by yuzzzdi** [huggingface/lerobot#2935]
### Ticket Type

🐛 Bug Report (Something isn't working)

### Environment & System Info

```Shell
ROS2 Humble, Ubuntu 22
```

### Description

Hi,

I'm deploying SmolVLA model to control a simulated Koch arm in Gazebo, I have trained the model on a hugging face dataset and I have configured the robot as per the training dataset.

My bridge works fine, camera data, joint states, prompt is being received by the model and the model predicts action output. The problem is that everytime the action output is same, It goes to a specific position and gets stuck. Even if I change the position of the block, the arm moves to the same position as per the models prediction.

I haven’t been able to find any resources related to this, so any help would be appreciated.

Thank you in advance 

<img width="965" height="843" alt="Image" src="https://github.com/user-attachments/assets/2fbea341-9779-47df-815b-5f550e96c158" />
<img width="1920" height="1173" alt="Image" src="https://github.com/user-attachments/assets/c8b9dab3-06b2-42d8-8425-2a8253ae9ba7" />

Example:

Loading weights: 100%|█| 489/489 [00:00<00:00, 679.76it/s, Materializing param=model.vision_model.post_lay
Reducing the number of VLM layers to 16 ...
Loading weights from local directory
Action chunk shape: torch.Size([1, 6])
Next action (deg): [ -2.3726625   82.05729    -22.306858   -81.84335      0.19299853
  18.817936  ]
Next action (rad): [-0.04141077  1.4321699  -0.38932812 -1.4284359   0.00336846  0.32843494]
Action chunk shape: torch.Size([1, 6])
Next action (deg): [ -2.13825     83.818405   -22.87389    -82.40293     -0.27328968
  15.588573  ]
Next action (rad): [-0.0373195   1.4629071  -0.3992247  -1.4382025  -0.0047698   0.27207193]
Action chunk shape: torch.Size([1, 6])
Next action (deg): [ -1.8797209   85.21373    -22.405865   -82.745094    -0.23670298
  13.73649   ]
Next action (rad): [-0.03280732  1.4872601  -0.39105612 -1.4441743  -0.00413125  0.23974699]
Action chunk shape: torch.Size([1, 6])
Next action (deg): [ -2.394196   86.51161   -22.549349  -83.32173    -1.1112518  12.8353405]
Next action (rad): [-0.0417866   1.5099125  -0.39356038 -1.4542385  -0.019395    0.22401895]
Action chunk shape: torch.Size([1, 6])
Next action (deg): [ -2.802764   88.18274   -23.674398  -83.93714    -1.8410969  12.435513 ]
Next action (rad): [-0.04891746  1.5390792  -0.4131962  -1.4649795  -0.0321332   0.21704064]
Action chunk shape: torch.Size([1, 6])
Next action (deg): [ -3.3034458  88.88332   -22.579296  -83.918655   -2.318234   12.766756 ]
Next action (rad): [-0.05765601  1.5513066  -0.39408305 -1.4646568  -0.04046081  0.22282192]

### Relevant logs or stack trace

```Shell

```

### Checklist

- [x] I have searched existing tickets to ensure this isn't a duplicate.
- [x] I am using the latest version of the `main` branch.
- [x] I have verified this is not an environment-specific problem.

**Comment by Nadslab** [comment]
@yuzzzdi What was the solution that you found?

**Comment by yuzzzdi** [comment]
@Nadslab I didn’t continue this, but I believe my setup didn’t work because the model's output is normalized to a range of [-100, 100]. I didn’t have a mapping to match the joint position range in Gazebo. I think it could work if there were a proper mapping.

**Comment by Nadslab** [comment]
Gotcha, thanks so much @yuzzzdi
